ee/tabby-ui/package.json

```diff
@@ -54,21 +56,26 @@
     "clsx": "^1.2.1",
     "compare-versions": "^6.1.0",
     "copy-to-clipboard": "^3.3.3",
+    "date-fns": "^3.6.0",
     "downshift": "^8.2.2",
     "filename2prism": "^3.0.0",
     "focus-trap-react": "^10.1.1",
     "graphql": "^16.8.1",
     "humanize-duration": "^3.31.0",
     "jwt-decode": "^4.0.0",
     "lodash-es": "^4.17.21",
+    "lucide-react": "^0.365.0",
     "mitt": "^3.0.1",
     "moment": "^2.29.4",
     "nanoid": "^4.0.2",
     "next": "^13.4.7",
     "next-themes": "^0.2.1",
+    "numeral": "^2.0.6",
     "openai-edge": "^0.5.1",
     "pretty-bytes": "^6.1.1",
     "react": "^18.2.0",
+    "react-activity-calendar": "^2.2.8",
+    "react-day-picker": "^8.10.0",
```

Comment:
```suggestion
```

Use https://ui.shadcn.com/docs/components/date-picker#date-range-picker

Reply:
This is needed, `date-range-picker` requires `calendar`, while `calendar` is built on `react-day-picker`